app/src/main/java/com/github/se/cyrcle/ui/navigation/NavigationActions.kt

```diff
@@ -31,8 +31,10 @@ data class TopLevelDestination(val route: String, val icon: ImageVector, val tex
 
 /** Object containing the top level destinations in the app. */
 object TopLevelDestinations {
-  val LIST = TopLevelDestination(route = Route.LIST, icon = Icons.Outlined.Menu, textId = "List")
-  val MAP = TopLevelDestination(route = Route.MAP, icon = Icons.Outlined.LocationOn, textId = "Map")
+  val LIST =
+      TopLevelDestination(route = Route.LIST, icon = Icons.Outlined.Menu, textId = Screen.LIST)
```

Comment:
I believe that if one wanted to use a predefined String for the textId, it would be better to opt for Route.LIST ("List") rateher than Screen.LIST ("List Screen"), in the context that this is the text that will be displayed on the buttons (or menu) that allow going from one TLD to another

Reply:
Yes thank you, I added the changes